lib/plugins.js

```diff
@@ -1,4 +1,3 @@
-const uuid = require('uuid')
```

Comment:
this filie is a complete mess because of the automatic formatter. undo all changes unrelated to the functionality